### PR TITLE
email api assign

### PR DESCRIPTION
**Pull request by onkar-josh**

### CODE REVIEW

File: emailapi/index.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+var express = require('express');
```
Comment (sunil4sonawane) [inline emailapi/index.js]: objects which we are not going to modify should be declared as const.

File: emailapi/index.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+var express = require('express');
+var app = express();
+
+var notification= require('./sendnotification.js');
+
+app.use('/email', notification);
+
+app.listen(3000);
```
Comment (sunil4sonawane) [inline emailapi/index.js:7]: Please check syntax.

File: emailapi/index.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+var express = require('express');
+var app = express();
+
+var notification= require('./sendnotification.js');
+
+app.use('/email', notification);
+
+app.listen(3000);
+console.log('server at 3000');
```
Comment (sunil4sonawane) [inline emailapi/index.js]: Add new line at file end

File: emailapi/mailsend.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
+var express = require('express');
+var router = express.Router();
+var nodemailer = require('nodemailer');
```
Comment (sunil4sonawane) [inline emailapi/mailsend.js]: follow camel case

File: emailapi/mailsend.js [hunk 5]
```diff
@@ -0,0 +1,31 @@
+var express = require('express');
+var router = express.Router();
+var nodemailer = require('nodemailer');
+var file= require('./sendnotification.js');
```
Comment (sunil4sonawane) [inline emailapi/mailsend.js]: file name should be in camel case only

File: emailapi/mailsend.js [hunk 6]
```diff
@@ -0,0 +1,31 @@
+var express = require('express');
+var router = express.Router();
+var nodemailer = require('nodemailer');
+var file= require('./sendnotification.js');
+
+
+  function senderfunction(mailOptions){
```
Comment (sunil4sonawane) [inline emailapi/mailsend.js]: code indentation should be followed with 2 space indent

File: emailapi/sendnotification.js [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+  subject: 'reguaring assignment status',
+  text: `assignment done successfully`
+};
+
+mailsender.senderfunction(mailOptions);
+
+res.json(mailOptions);
+});
+
+module.exports = router;
```
Comment (sunil4sonawane) [inline emailapi/sendnotification.js]: Add new line at end

File: emailapi/mailsend.js (hunk 6 above)
Comment (sunil4sonawane) [inline emailapi/mailsend.js]: camel case should be followed

**Review by sunil4sonawane — CHANGES_REQUESTED** [review]
Please go through comments and do necessary corrections

File: emailapi/mailsend.js [hunk 8]
```diff
@@ -0,0 +1,26 @@
+const express = require('express');
+const router = express.Router();
+const nodeMailer = require('nodemailer');
+const file= require('./sendNotification.js');
+
+function senderFunction(mailOptions){
+	var transporter = nodeMailer.createTransport({
+	service: 'gmail',
```
Comment (sunil4sonawane) [inline emailapi/mailsend.js:8]: code indentation is incorrect

**Review by sunil4sonawane — CHANGES_REQUESTED** [review]
Do code indentation and push the code

File: emailapi/mailsend.js (hunk 8 above)
Comment (onkar-josh) [reply emailapi/mailsend.js:8]: I have done it in my file but changes are not reflecting over here.

**Pull request closed without merging**